CHANGELOG.md

```diff
@@ -1,3 +1,26 @@
+1.4.0 / 2019-08-28
+==================
+
+###### Changes
+* Dropping support for PHP 5.3
```

Comment:
Please move into `Testsuite` sub-section of changelog and change to:

```
* Don't test on PHP 5.3 (driver itself would likely continue to work on PHP 5.3 for some time)
```

---

We haven't dropped support for PHP 5.3 in code, but only in test suite.

Reply:
php version was bumped also in this package.